src/persist-client/src/internal/diff.proto

```diff
@@ -20,7 +20,10 @@ enum ProtoStateField {
     CRITICAL_READERS = 6;
     WRITERS = 3;
     SINCE = 4;
-    SPINE = 5;
+    LEGACY_BATCHES = 5;
+    HOLLOW_BATCHES = 9;
+    SPINE_BATCHES = 10;
+    SPINE_MERGES = 11;
```

Comment:
> I think it would be good to have it ignore any entries with unrecognized diff types, in the same way decoding a rollup will ignore unrecognized features from the future.

Yup that seems reasonable